SeaPublicWebsite.Data/SeaPublicWebsite.Data.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
```

Comment:
As we're not using nullable reference types in the main project I think it's going to be confusing to use them just in the data project - especially as EF doesn't seem to play very nicely with nullable reference types anyway: https://docs.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types
This will also get rid of the build warning on `SeaDbContext.cs`

Reply:
Done :)